tests/unit/generic_agent_test.c

```diff
@@ -42,14 +42,15 @@ void test_have_tty_interactive_failsafe_is_not_created(void)
 
     PolicyDestroy(policy);
     GenericAgentFinalize(ctx, config);
+    CryptoDeInitialize();
```

Comment:
There are two additions of calling `CryptoDeInitialize()` that aren't fixing whitespace. We need an explanation of those additions in the commit message I think.